src/components/LoginMapButton.js

```diff
@@ -0,0 +1,10 @@
+import React, { PureComponent } from 'react';
+import './LoginMapButton.css';
+
+export default class LoginMapButton extends PureComponent {
+  render() {
+    return (
+      <div className="LoginMapButton-btn"/>
```

Comment:
And then have an onClick handler on your div that calls the onClick function passed to the Component via its props via:
`<div onClick={onClick} className="LoginMapButton-btn"/>`

Now the click should work ;)